Trackovid19-web/src/app/shared/share-status/share-status.component.html

```diff
@@ -46,9 +47,23 @@
           >
             <img src="assets/img/icons/fb-share-icon.svg" />
           </div>
-          <div *ngIf="video.share.twitter" id="twitter" class="icon-share" (click)="shareTwitter()">
-            <img src="assets/img/icons/twitter-share-icon.svg" />
+
+          <!-- Whatsapp -->
+          <div *ngIf="video.share" id="whatsapp" class="icon-share" (click)="shareWhatsapp()">
+            <img src="assets/img/icons/whatsapp-share-icon.svg" />
           </div>
+
+          <!-- Facebook messenger -->
+          <div
+            *ngIf="video.share"
```

Comment:
Can you update all the new 3 icons to check if is to show like the others?
For instance this one: 
```
*ngIf="video.share.messenger"
```
we will update the API now to use this codes:

- messenger
- whatsapp
- copyText

Reply:
Yes! i do right now!